openpype/modules/deadline/plugins/publish/submit_publish_job.py

```diff
@@ -294,12 +297,13 @@ def _submit_deadline_post_job(self, instance, job, instances):
                 job_index += 1
         elif instance.data.get("bakingSubmissionJobs"):
             self.log.info("Adding baking submission jobs as dependencies...")
+
             job_index = 0
             for assembly_id in instance.data["bakingSubmissionJobs"]:
                 payload["JobInfo"]["JobDependency{}".format(
                     job_index)] = assembly_id  # noqa: E501
                 job_index += 1
-        else:
+        elif job.get("_id"):
```

Comment:
Should we add an `else:` branch doing a debug log saying "No dependent jobs detected." just to make sure we can trace it nicely if start hitting the else branch?